recipes/pantable/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pantable" %}
+{% set version = "0.12.4" %}
```

Comment:
Looks like 0.13.1 was just release, which includes the `LICENSE` file in the tarball on PyPI.

Reply:
Yes, I’m the author of the package and want to push the older version first, which supports much older versions of pandoc, for compatibility reasons.

Reply:
Fair enough.